Makefile

```diff
@@ -1,6 +1,9 @@
-EXECUTABLE=dns-controller-manager
-PROJECT=github.com/gardener/external-dns-management
-VERSION=$(shell cat VERSION)
+REGISTRY              := gcr.io/pingcap-public/gardener
+EXECUTABLE            := dns-controller-manager
+PROJECT               := github.com/gardener/external-dns-management
```

Comment:
I'm wondering if this need to be adjusted to the forking repo, but it seems this is not used anywhere.

Reply:
Yes, we don't need to update this